### PR TITLE
[FIX][POS] Set analytic account on tax's move line

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Hi,

it's possible to define an analytic account on a tax's object.
But currently, when you generate the accounting entries at the closure of a POS session, taxes entries does not take analytic account defined on tax's object previously.

Changes in  b30d1f8  fix this behavior by adding analytic account defined on tax in the move line.

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
@fvdsn What do you think about this one ?

**Comment by adrienpeiffer** [comment]
OPW 632164

**Comment by adrienpeiffer** [comment]
@fvdsn Any news about this one ?

**Comment by beledouxdenis** [comment]
This issue doesn't happen in the Odoo 7.0 point of sale ? Shouldn't this be done against 7.0 instead ?

**Comment by adrienpeiffer** [comment]
Hi Denis,
Thanks for your answer !
I don't really know. Our custoimer work with 8.0 and therefore I tested and have provided a patch for this version.

**Comment by beledouxdenis** [comment]
I believe the fix is correct, but the pull request should be against 7.0.

**Comment by adrienpeiffer** [comment]
So, can you merge this commit in 7.0 and forward port it on 8.0 ?

File: addons/point_of_sale/point_of_sale.py [hunk 1]
```diff
@@ -1132,7 +1132,7 @@ def insert_data(data_type, values):
 
                 for tax in computed_taxes:
                     tax_amount += cur_obj.round(cr, uid, cur, tax['amount'])
-                    group_key = (tax['tax_code_id'], tax['base_code_id'], tax['account_collected_id'], tax['id'])
+                    group_key = (tax['tax_code_id'], tax['base_code_id'], tax['account_collected_id'], tax['id'], tax['account_analytic_collected_id'])
```
Comment (adrienpeiffer) [inline addons/point_of_sale/point_of_sale.py]: On further consideration, this line seems not necessary

**Comment by beledouxdenis** [comment]
Yes, as soon as this pull request is done against 7.0.

**Comment by C3POdoo** [comment]
Hello,

This is an automatic message.

Thank you for your contribution but the version 8.0 is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.

If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards,

**Pull request closed without merging**